### ISSUE TITLE
Retrieving keys named without leading slash

### ISSUE THREAD

**Issue by kemko** [kelseyhightower/confd#657]
Keys in etcdv3 may start without leading slash. Stolon for example keep their data with prefix `stolon/cluster/stolon-cluster`. I'm tried various confd settings to retrieve key stolon/cluster/stolon-cluster/clusterdata but nothing successed. On the other hand with keys started with `/` confd working with no any problems.

Maybe I did something wrong?

conf.d/test.toml:
```
[template]
src = "test.tmpl"
dest = "/tmp/test.conf"
keys = [
  "stolon/cluster/stolon-cluster/clusterdata",
]
```

templates/test.tmpl:
```
[myconfig]
{{getv "stolon/cluster/stolon-cluster/clusterdata"}}
```

**Comment by okushchenko** [comment]
Confd always prepends a `/` to all keys to make things consistent. Changing this might cause a huge regression, so should be carefully executed, but that is something that should definitely be done.

**Comment by hubo1016** [comment]
Maybe an option should be added to allow to retrieve keys without the first slash, or even allow customized  separators.

**Comment by okushchenko** [comment]
@hubo1016 agree, there is an option for a customized separator already, but it's only used by the new Redis backend implementation.

**Comment by stawii** [comment]
I suggest that keys WITH leading `/` are absolute and those without are relative to prefix.
By this one can easy set `stolon/cluster/stolon-cluster` as a prefix, and use `clusterdata` to receive `stolon/cluster/stolon-cluster/clusterdata` but `/something/other/data` will be absolute/full path (if accepted by `keys` whitelist).
